### PR TITLE
WIP tests AfterSuite: Force namespaces deletion

### PR DESCRIPTION
**Pull request by ormergi**
**What this PR does / why we need it**:
This PR change the way AfterSuite cleansup test namespaces, forcing deletion by removing finelizers.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/utils.go [hunk 1]
```diff
@@ -1721,25 +1725,50 @@ func cleanNamespaces() {
...
+			}
 		}
 	}
 
-	// Wait until the namespaces are terminated
+	// Force namesapce deletion by removing Finalizers
+	for _, namespace := range testNamespaces {
+		currentNamespace, err := virtCli.CoreV1().Namespaces().Get(namespace, metav1.GetOptions{})
+		if err != nil {
+			log.Log.Errorf("Could not get namespace in order to remove Finalizers %v", err)
```
Comment (danielBelenky) [inline tests/utils.go:1748]: If you were not able to get the namespace then you should exit here... you'll get a NPE.

File: tests/utils.go [hunk 2]
```diff
@@ -1721,25 +1725,50 @@ func cleanNamespaces() {
...
+		currentNamespace.Finalizers = []string{}
+
+		result := virtCli.RestClient().
+			Post().
+			RequestURI(fmt.Sprintf("/api/v1/namespaces/%s/finalize", currentNamespace.Name)).
+			Body([]byte(currentNamespace.String())).
+			Do()
+
+		if result.Error() != nil {
+			log.Log.Errorf("Could not delete namespace %v", result.Error().Error)
```
Comment (danielBelenky) [inline tests/utils.go:1759]: The error message here is incorrect. You was not able to set finalizers...

File: tests/utils.go (hunk 1 above)
Comment (ormergi) [reply tests/utils.go:1748]: If a namespace could not be found at this time, it means it already deleted

Comment (ormergi) [reply tests/utils.go:1748]: Now I see it, I will fix it in next commit

File: tests/utils.go (hunk 2 above)
Comment (ormergi) [reply tests/utils.go]: You right I will fix it in next commit

**Comment by ormergi** [comment]
/close
Another try at #3797

**Pull request closed without merging**